src/rust/engine/process_execution/src/remote.rs

```diff
@@ -269,7 +269,7 @@ impl super::CommandRunner for CommandRunner {
             let command_runner = command_runner.clone();
             let build_id = context.build_id.to_owned();
             move |summary| {
-              history.current_attempt += summary;
+              history.current_attempt += &summary;
```

Comment:
If this continued to pass by value, could we avoid cloning in `AddAssign` ?

Reply:
Good call - done!